compiler/luci/pass/src/QuantizationUtils.cpp

```diff
@@ -315,4 +315,85 @@ uint32_t cal_offset(loco::TensorShape &dimension, uint32_t *indices)
          indices[2] * dimension.dim(3).value() + indices[3];
 }
 
+ActivationQType activation_qtype(const CircleNode *node)
+{
+  auto fused_act_node = dynamic_cast<const CircleNodeMixin<CircleNodeTrait::FusedActFunc> *>(node);
+  if (fused_act_node && fused_act_node->fusedActivationFunction() == FusedActFunc::TANH)
+    return ActivationQType::PreDefinedValue;
```

Comment:
This is not covered at below `case` statement?

Reply:
This code checks fused activation function. For example, Conv + Tanh or DConv + Tanh.

The below code checks node itself is Tanh.